swap_meet/clothing.py

```diff
@@ -0,0 +1,14 @@
+from .item import Item
+
+class Clothing(Item):
+    
+    def __init__(self, category = "", condition=0):
+        self.category = "Clothing"
+        self.condition = condition
```

Comment:
this works! But since you are using inheritance by importing `Item` and passing it in, we can take advantage of the parent class's `__init__` method that's already been constructed.

```suggestion
class Clothing(Item):
    def __init__(self, condition=0): # because we know Clothing class will always be category "clothing", we don't need to give the user the ability to pass in a value
        super().__init__("clothing", condition) # here we are calling the parent class's __init__ method and we pass in the argument from the Child class down to the parent with "clothing". Now we don't need to assign self.category or self.condition again 
```

here's another way we could do it based on Learn's `super()` lesson:
```suggestion
class Clothing(Item):
    # if we don't change the parameters and keep them exactly like the parent class's __init__ method, then we don't technically have to create a child __init__ method because it will inherit the parent's. So only the __str__ method will be in this class
```

Both options work, but I would say the top suggestion is better practice, and you will see it in industry